scripts/integration/mqtt/test.yaml

```diff
@@ -0,0 +1,12 @@
+features:
```

Comment:
I think we need to run these tests to CI. That would mean adding to https://github.com/vectordotdev/vector/blob/master/.github/workflows/integration.yml . You can see the other tests there for an example.

Reply:
good catch, added in https://github.com/vectordotdev/vector/pull/19813/commits/7dbb96edf54729e4a1bc08097885e3bf392acd54